pages/linux/pacman.md

```diff
@@ -27,9 +27,9 @@
 
 `pacman -Qe`
 
-- Find which package owns a certain file:
+- List orphan dependencies:
 
-`pacman -Qo {{filename}}`
+`pacman -Qtdq`
```

Comment:
```suggestion
`pacman --query --unrequired --deps --quiet`
```

Reply:
Shouldn't we change this for all of the commands here?

because that would make the command `sudo pacman -Syu` something which would probably be harder to remember than Syu

Reply:
Yeah, I think we should, all the subcommands are in long form.